bolt/internal/internal_test.go

```diff
@@ -133,8 +139,8 @@ func TestMarshalLayout(t *testing.T) {
 		t.Fatal(err)
 	} else if err := internal.UnmarshalLayout(buf, &vv); err != nil {
 		t.Fatal(err)
-	} else if !reflect.DeepEqual(layout, vv) {
-		t.Fatalf("source protobuf copy error: got %#v, expected %#v", vv, layout)
+	} else if !cmp.Equal(layout, vv) {
+		t.Fatal("source protobuf copy error: diff:\n", cmp.Diff(layout, vv))
```

Comment:
interesting approach with cmp.Diff. seems slightly tautological; this is really just testing that the marshal and unmarshal works as intended... which, won't it?